diff --git a/AmberTools/src/packmol_memgen/CMakeLists.txt b/AmberTools/src/packmol_memgen/CMakeLists.txt
index 26eff51cdb..d79e26963a 100644
--- a/AmberTools/src/packmol_memgen/CMakeLists.txt
+++ b/AmberTools/src/packmol_memgen/CMakeLists.txt
@@ -4,7 +4,7 @@ project(packmol_memgen)
 
 # first, build the three native executables it uses
 add_subdirectory(packmol_memgen/lib/memembed)
-add_subdirectory(packmol_memgen/lib/packmol)
+# add_subdirectory(packmol_memgen/lib/packmol)
 add_subdirectory(packmol_memgen/lib/ppm3)
 
 # --------------------------------------------------------------------
